CMakeLists.txt

```diff
@@ -22,6 +22,28 @@ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake")
 include(ExternalProject)
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wno-comment -Wno-sign-compare")
 
+SET(CMAKE_CXX_FLAGS_SANITIZE
```

Comment:
Can you please put a comment block indicating what this flag does and how to use it? Thanks.

Reply:
:+1: